src/ProgressOnderwijsUtils/ProgressOnderwijsUtils.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\NugetPackagesCommon.props" />
   <PropertyGroup Label="Configuration">
-    <Version>70.0.0</Version>
-    <PackageReleaseNotes>Update to .NET 5.</PackageReleaseNotes>
+    <Version>71.0.0</Version>

```

Comment:
Het zou eigenlijk 70.1.0 zijn denk ik, want je hebt alleen iets toegevoegd toch?

Reply:
ja alleen iets toegevoegd. Ik heb geen idee van de regels hieromtrent. Had ik die ergens kunnen vinden? Ik zal het aanpassen

Reply:
Major.Minor.Version:
Major is als er iets verwijderd veranderd is, dus niet bacwards compatible.
Minor is als er iets backwards compatible bij komt.
Version is als er alleen een implementatie is aangepast, maar niet de api.